docs/querying/basics.md

```diff
@@ -60,21 +58,20 @@ Scalar float values can be literally written as numbers of the form
 
     -2.43
 
-## Time series Selectors
+## Time series selectors
 
 ### Instant vector selectors
 
 Instant vector selectors allow the selection of a set of time series and a
-single sample value for each at a given timestamp (instant): in the simplest
+single sample value for each at a given timestamp (instant). In the simplest
 form, only a metric name is specified. This results in an instant vector
 containing elements for all time series that have this metric name.
 
-This example selects all time series that have the `http_requests_total` metric
-name:
+This example selects all time series that have the `http_requests_total` metric name:
 
     http_requests_total
 
-It is possible to filter these time series further by appending a comma separated list of label
+You can filter time series further by appending a comma separated list of label
```

Comment:
Don't change the style of the text

Reply:
"Reference documentation is about describing a piece of software, so we should be talking about the software in the 3rd person rather than talking to the user in the 2nd person. "it is possible to" would be unclear for reference documentation, I don't want to know some things that might be possible - I want to know exactly how the software behaves." - Brian Brazil

Would you prefer "Prometheus allows you to"?

Reply:
I'd personally go with something like "Time series can be further filtered by"